pkg/coreapi/graph/resolvers/event.resolver.go

```diff
@@ -90,16 +90,7 @@ func (r *eventResolver) Status(ctx context.Context, obj *models.Event) (*models.
 }
 
 func (r *eventResolver) Raw(ctx context.Context, obj *models.Event) (*string, error) {
-	// Runner.Events maps events by an event ID that could be either the
-	// internal or external ID. It'll be the internal ID if the event didn't
-	// have an external ID
```

Comment:
As of this change, I believe this comment is no longer true: https://github.com/inngest/inngest/pull/1238/files#diff-52e41636f29e1f855e51f1ad1144dd1cc18e411574f06628ddc711f2564130d5R118 cc @jpwilliams for confirmation.